src/assets/styles/10-settings/_colours.scss

```diff
@@ -21,20 +21,21 @@
   --colour-cyan20: #beebf4;
   --colour-cyan05: #f1fcfd;
 
-  --colour-concrete-90: #000;
-  --colour-concrete-80: #292929;
-  --colour-concrete-70: #5c5c5c;
-  --colour-concrete-60: #767676;
-  --colour-concrete-50: #878787;
-  --colour-concrete-40: #999;
-  --colour-concrete-30: #b3b3b3;
-  --colour-concrete-20: #ccc;
-  --colour-concrete-10: #e6e6e6;
-  --colour-concrete-05: #f2f2f2;
+  --colour-grey-90: #000;
```

Comment:
Latest version of Wellcome Design System uses grey instead of concrete - https://zpl.io/2GWJzlW

Reply:
Ah that's annyoing - there are two colour palettes in Zeplin https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5d1b8ba81a1c765706ccbcc1